pkg/webhook/webhook.go

```diff
@@ -322,3 +353,19 @@ func (c *Webhook) validateApplication(request *admission.AdmissionRequest, appli
 
 	return err
 }
+
+func (c *Webhook) validateObject(obj metav1.Object) error {
```

Comment:
Could the function name mention something about blocks? Or this method being pushed to the `rolloutblock` package? As per the signature, it does not feel like it would only check for block related things.